code/src/css/moviecards.css

```diff
@@ -0,0 +1,72 @@
+.movie-cards {
+    display: grid;
+    width: 100%;
+    height: 100%;
+  }
+  
+  .movie-cards a {
+    position: relative;
+    color: #fff;
+    text-decoration: none;
+    width: 100%;
+    height: 100%;
+  }
+  
+  .movie-poster {
+    width: 100%;
+    height: 100vh;
```

Comment:
I would remove the height for the posters because they get a little disfigured on smaller screens when the height is 100vh

Reply:
I also think you can remove the heights for the media queries since they will adjust according to the widths which is 100%